website/content/docs/commands/plugin/register.mdx

```diff
@@ -24,17 +24,17 @@ Success! Registered plugin: my-custom-plugin
 
 Register an Enterprise plugin:
 
--> **Note:** See [Plugin managment for Enterprise plugins](/vault/docs/plugins/plugin-management#enterprise-plugins)
+-> **Note:** See [Plugin management for Enterprise plugins](/vault/docs/plugins/plugin-management#enterprise-plugins)
```

Comment:
We shouldn't be adding an aside between instructions and the intro to those instructions. I would recommend either ditching the aside and making "an Enterprise plugin" in the previous sentence the link anchor or moving the aside to *after* the Enterprise example code.

If we decide to keep the aside:

```suggestion
<Note>

Refer to the [Enterprise plugins](/vault/docs/plugins/plugin-management#enterprise-plugins) section
of the Plugin management page for Vault compatibility requirements.

</Note>
```